app/javascript/packs/sdk.js

```diff
@@ -18,7 +18,8 @@ const runSDK = ({ baseUrl, websiteToken }) => {
   }
 
   const chatwootSettings = window.chatwootSettings || {};
-  let locale = chatwootSettings.locale || 'en';
+  let locale = chatwootSettings.locale;
```

Comment:
The bug was happening as `App.vue` will set the locale from the chatwoot account correctly. But the widget sets 'en' as the default language and then updates locale on `App.vue` through postMessage. Which should not happen if no language is set by user in `window.chatwootSettings`